### PR TITLE
add master_ssl_verify_identity to mysql_replication

### PR DESCRIPTION
**Pull request by chrnie**
I've added master_ssl_verify_identity parameter to mysql_replication module.

Reason: I need this, MySQL/mariaDB provides it. Ansible should support this.

### CODE REVIEW

File: lib/ansible/modules/database/mysql/mysql_replication.py [hunk 1]
```diff
@@ -84,6 +84,11 @@
     master_ssl_cipher:
         description:
             - same as mysql variable
+    master_ssl_verify_identity:
+        description:
+            - activate host name identity verification
+        default: 0
+        choices: [ 0, 1 ]
```
Comment (gundalow) [inline lib/ansible/modules/database/mysql/mysql_replication.py:91]:
```
        type: bool
        version_added: '2.10'
```

File: lib/ansible/modules/database/mysql/mysql_replication.py [hunk 2]
```diff
@@ -84,6 +84,11 @@
     master_ssl_cipher:
         description:
             - same as mysql variable
+    master_ssl_verify_identity:
+        description:
+            - activate host name identity verification
+        default: 0
```
Comment (gundalow) [inline lib/ansible/modules/database/mysql/mysql_replication.py:90]: default: False

**Review by gundalow — CHANGES_REQUESTED** [review]

**Comment by gundalow** [comment]
needs_info # Will auto close this PR if it doesn't get updated

**Comment by Andersson007** [comment]
@chrnie , could you please
1. cover this by integration tests (there aren't existent tests for mysql_replication, so it needs to be added). We need to be sure that this doesn't break anything. I suspect that will be a challange. Probably unit tests would be enough.
2. fix points that @gondalow requested
3. add an example of using this parameter

**Comment by Andersson007** [comment]
@bmalynovytch what do you thing about this option?

**Comment by gundalow** [comment]
Given PR this was a first time contributor, and the branch (fork) has gone away I don't think we'll see @chrnie return to this, so I think this can be closed.

**Comment by Andersson007** [comment]
@gundalow , i think, yes.
@chrnie , if you fix all of requests above, please, just open a new PR, i'll review it (ping me to aaklychkov at mail dor ru)

**Comment by gundalow** [comment]
Given the above, I'll close this.

**Pull request closed without merging**